### PR TITLE
[FEATURE] #102935 - Add PackageInitializationEvent

### PR DESCRIPTION
**Pull request by brotkrueml**
Additionally, four events from the extension manager have been removed (only three were documented).

Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/833
Releases: main

### CODE REVIEW

**Comment by brotkrueml** [comment]
I opted for not setting the pages to orphan, otherwise I have to reference the remaining event explicitly in the toc of the parent page - which would be error-prone when additional events for this extension would be added.

File: Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+allows listeners to execute custom functionality after a package has been
+activated.
+
+The event is being dispatched at several places, where packages get activated.
+Those are, for example:
+
+*   on extension installation by the extension manager
+*   on calling the :bash:`typo3 extension:setup` command.
+
+However, the main component, dispatching the event, is the
```
Comment (garvinhicking) [inline Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst:21]: "However" seems to counteract the examples before, maybe not needed?

```suggestion
The main component, dispatching the event, is the
```

**Review by garvinhicking — APPROVED** [review]
Minor remarks, good to go generally :)

File: Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+==========================
+
+..  versionadded:: 13.0
+
+The PSR-14 event :php:`\TYPO3\CMS\Core\Package\Event\PackageInitializationEvent`
+allows listeners to execute custom functionality after a package has been
+activated.
+
+The event is being dispatched at several places, where packages get activated.
+Those are, for example:
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst:16]: As installed Composer packages are activated automatically, yes, should be. Will replace "package" with "extension".

File: Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+..  _PackageInitializationEvent:
+
+==========================
+PackageInitializationEvent
+==========================
+
+..  versionadded:: 13.0
+
+The PSR-14 event :php:`\TYPO3\CMS\Core\Package\Event\PackageInitializationEvent`
+allows listeners to execute custom functionality after a package has been
```
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst]: Done

File: Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (brotkrueml) [reply Documentation/ApiOverview/Events/Events/Core/Package/PackageInitializationEvent.rst:16]: Done

**Review by garvinhicking — APPROVED** [review]
👍

**Pull request merged**